openmetadata-ui/src/main/resources/ui/src/utils/Users.util.tsx

```diff
@@ -145,3 +146,7 @@ export const commonUserDetailColumns = (
     },
   },
 ];
+
+export const isMaskedEmail = (email: string) => {
+  return email === MASKED_EMAIL;
```

Comment:
Hello @chirag-madlani , I was wondering why we have equal checks here.
masked email value is ` '********@masked.com';` , for different user it will be different I belive

Reply:
@Sachin-chaurasiya it's hardcoded 

<img width="1180" alt="Screenshot 2024-04-22 at 4 54 46 PM" src="https://github.com/open-metadata/OpenMetadata/assets/12962843/d2c66ffe-933d-4984-9c95-39252f6eb94e">